modules/social_features/social_event/social_event.module

```diff
@@ -422,13 +459,13 @@ function _social_event_format_date($event, $view_mode) {
 function social_event_social_user_account_header_create_links($context) {
   return [
     'add_event' => [
-      '#type' => 'link',
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.